extensions/s3/src/main/java/io/deephaven/extensions/s3/S3Instructions.java

```diff
@@ -24,12 +24,23 @@
 @CopyableStyle
 public abstract class S3Instructions implements LogOutputAppendable {
 
-    private final static int DEFAULT_MAX_CONCURRENT_REQUESTS = 256;
-    private final static int DEFAULT_READ_AHEAD_COUNT = 32;
-    private final static int DEFAULT_FRAGMENT_SIZE = 1 << 16; // 64 KiB
-    private final static int MIN_FRAGMENT_SIZE = 8 << 10; // 8 KiB
-    private final static Duration DEFAULT_CONNECTION_TIMEOUT = Duration.ofSeconds(2);
-    private final static Duration DEFAULT_READ_TIMEOUT = Duration.ofSeconds(2);
+    private static final int DEFAULT_MAX_CONCURRENT_REQUESTS = 256;
+    private static final int DEFAULT_READ_AHEAD_COUNT = 32;
+    private static final int DEFAULT_FRAGMENT_SIZE = 1 << 16; // 64 KiB
+    private static final int MIN_FRAGMENT_SIZE = 8 << 10; // 8 KiB
+    private static final Duration DEFAULT_CONNECTION_TIMEOUT = Duration.ofSeconds(2);
+    private static final Duration DEFAULT_READ_TIMEOUT = Duration.ofSeconds(2);
+    private static final int DEFAULT_NUM_CONCURRENT_PARTS = 64;
+
+    /**
+     * We set maximum part size to 10 MB. The maximum number of parts allowed is 10,000. This means maximum size of a
+     * single file that we can write is roughly 100k MB (or about 98 GB). For uploading larger files, user would need to
+     * set a larger part size.
+     *
+     * @see <a href="https://docs.aws.amazon.com/AmazonS3/latest/userguide/qfacts.html">Amazon S3 User Guide</a>
+     */
+    private static final int MIN_PART_SIZE_MB = 5;
+    private static final int DEFAULT_PART_SIZE_MB = 10;
```

Comment:
We should be consistent either using `MB` everywhere, or `MiB` / `MIB` everywhere.

Reply:
(Ditto w/ GB vs GiB, etc)